addons/mail/static/src/models/channel_invitation_form/channel_invitation_form.js

```diff
@@ -25,7 +25,7 @@ function factory(dependencies) {
          * Closes this channel invitation form.
          */
         close() {
```

Comment:
close method becomes kinda obsolete, with the new modelling "close" should always be just delete